### ISSUE TITLE
Linux build errors 

### ISSUE THREAD

**Issue by git-bone** [valhalla/valhalla#3617]
I'm having trouble building from source on linux (debian testing, fully up to date). The error message looks quite similar to this one: https://github.com/valhalla/valhalla/issues/3388. Reverting to libgeos version 3.9 with its dependencies does not solve the build error. 

Are there any workarounds I can try?

```
[ 86%] Linking CXX executable valhalla_build_admins
/usr/bin/ld: src/libvalhalla.a(adminbuilder.cc.o): in function `valhalla::mjolnir::BuildAdminFromPBF(boost::property_tree::basic_ptree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&)':
adminbuilder.cc:(.text+0x3e3d): undefined reference to `geos::geom::Coordinate::Coordinate()'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/valhalla_build_admins.dir/build.make:108: valhalla_build_admins] Error 1
make[2]: Leaving directory '/valhalla/build'
make[1]: *** [CMakeFiles/Makefile2:1568: CMakeFiles/valhalla_build_admins.dir/all] Error 2

```

**Comment by kevinkreiser** [comment]
its not yet ready but @nilsnolde is working on removing this type of problem as an obstacle completely by porting all of the geometry stuff to the boost geometry api #3396 #3616 

as for reverting and trying a different version, im using 3.9.1 on ubuntu which shouldnt be too terribly different from what you have on debian. are you sure you are actually linking against that installed version and not against a newer one? if you make with VERBOSE=1 you should get the full path to the library its trying to link

**Comment by git-bone** [comment]
Let me try again with verbose=1. I got all kinds of errors on geos but it just might be looking in the wrong location.

**Comment by git-bone** [comment]
ok it works now. It seems libgeos++-dev was removed when I removed libgeos that was installed from the repos. I installed it manually from stable branch and now I can build succesfully.
